src/legacy/core_plugins/kibana/public/visualize/np_ready/editor/editor.js

```diff
@@ -372,22 +389,22 @@ function VisualizeAppController(
     $scope.timeRange = timefilter.getTime();
     $scope.opts = _.pick($scope, 'savedVis', 'isAddToDashMode');
 
-    stateMonitor = stateMonitorFactory.create($state, stateDefaults);
-    stateMonitor.ignoreProps(['vis.listeners']).onChange(status => {
-      $appStatus.dirty = status.dirty || !savedVis.id;
-    });
+    const stateContainerSubscription = stateContainer.state$.subscribe(state => {
```

Comment:
Nit: You could just do it that way: `stateContainer.subscribe(`

Reply:
oh, sure, thanks!
I just followed docs here:
`src/plugins/kibana_utils/docs/state_containers/no_react.md`

Which says:

![image](https://user-images.githubusercontent.com/31325372/74428300-cad96980-4e69-11ea-8ede-847900c5db8f.png)

Seems docs should be updated =)

Reply:
Seems there are several roads that lead to Rome :)